profiles/scalingo.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "Scalingo",
-  "revision": "2015-07-22",
-  "vendor_verified": "2015-04-17",
+  "revision": "2016-08-04",
+  "vendor_verified": "2015-02-23",
```

Comment:
`vendor_verified` means the last date an official member of the provider reviewed the profile. So this should be today?!